service/service.go

```diff
@@ -52,6 +52,7 @@ func New(ctx context.Context, config config.Config, serviceLogger *logging.Servi
 
 	// create an http router for registering handlers for a given route
 	mux := http.NewServeMux()
+	registerPprofHandlers(config, mux)
```

Comment:
can you add an env variable flag for enabling/disabling this registration?

Reply:
ah nice yeah good point